controllers/operandrequest/reconcile_operand.go

```diff
@@ -130,15 +130,33 @@ func (r *Reconciler) reconcileOperand(ctx context.Context, requestInstance *oper
 				continue
 			}
 
-			if csv.Status.Phase == olmv1alpha1.CSVPhaseFailed {
-				merr.Add(fmt.Errorf("the ClusterServiceVersion of Subscription %s/%s is Failed", namespace, operatorName))
-				requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorFailed, "", &r.Mutex)
+			if sub.Status.Install == nil || sub.Status.InstallPlanRef.Name == "" {
+				klog.Warningf("The Installplan for Subscription %s is not ready. Will check it again", sub.Name)
 				continue
 			}
-			if csv.Status.Phase != olmv1alpha1.CSVPhaseSucceeded {
-				klog.Errorf("the ClusterServiceVersion of Subscription %s/%s is not Ready", namespace, operatorName)
-				requestInstance.SetMemberStatus(operand.Name, operatorv1alpha1.OperatorInstalling, "", &r.Mutex)
-				continue
+
+			// If the installplan is deleted after is completed, ODLM won't block the CR update.
```

Comment:
I recommend that we could add an additional comment here, talking about the scenario where InstallPlan is not generated yet is handled by checking `sub.Status.Install == nil` above in line 133 for further reference.

Reply:
I add the comment in line 133 since 133 is for checking if IP exist. Is it ok？